README.md

```diff
@@ -1,13 +1,32 @@
+[Subsquid ](https://subsquid.io)is a query node framework for Substrate-based blockchains. A query node ingests data from a substrate chain and provides rich, domain-specific, and highly customizable access to the blockchain data, far beyond the scope of direct RPC calls.
+
+For example, expired [Kusama Treasury](https://wiki.polkadot.network/docs/en/learn-treasury) spending [proposals](https://kusama.subscan.io/event?module=Treasury\&event=Proposed) are pruned from the state of the [Kusama blockchain](https://polkascan.io/kusama), so querying, say, one-year-old proposals is problematic. Indeed, one has to track the evolution of the state by sequentially applying the Treasury events and [extrinsics ](key-concepts/extrinsic-in-substrate.md)in each historical block.
 
 That's where Subsquid gets you covered. Define your data model and the Squid Archive will get it in sync with the chain. On top of that, you get a batteries-included GraphQL server with comprehensive filtering, pagination, and even full-text search capabilities.
+
+### Data retrieval made ~~easy~~ easier :smile:
+
+Blockchains produce, add, and store data in a very different way to centralized sources, and therefore the ways in which we must search for it is different too. The beauty of blockchain is that anyone can tap into the data held within it to create their own DApp, but this has led to a morass of data spread across blockchains that has never been utilized.
+
+Poor data leads to poor products, and before we know it, blockchain solutions are failing because the data feeds are simply not up to the job. We have barely begun to scratch the surface of what can be achieved in a decentralized world.
+
+This is where Subsquid comes in. Subsquid has a multi-layered [architecture](key-concepts/architecture.md), composed of [Squid Archive(s)](key-concepts/architecture.md#squid-archive) and [Squid(s)](key-concepts/architecture.md#squid).
+
+Taking the example of Polkadot ecosystem, which is the first of many blockchains Subsquid will serve: it includes several `parachains` which are used by different projects that require very different datasets. This means that the type of data moving around the Polkadot ecosystem is not only large in volume, but also in variety.&#x20;
```

Comment:
I think we can remove this paragraph?

Reply:
Took it from our [Reddit post](https://www.reddit.com/r/subsquid/comments/pglb96/subsquid_the_hatching/), paraphrased it a little bit. You mean the part about Polkadot and parachains, right?

Reply:
Yeah it doesn't feel like it adds anything new here from my perspective. Would like to have the docs to be more concise and less fluffy (this only concerns the polkadot part).